packages/sdk-core/src/modules/storage/SessionFiles.ts

```diff
@@ -5,15 +5,19 @@ import { FileSystem } from './FileSystem.js';
 
 interface FileSession {
     readonly file: string;
-    readonly sessionId: string;
+    readonly sessionId: SessionId;
```

Comment:
how about backward compatibility?

Reply:
You're right, this could fail. I added a check for this where required.